.eslintrc.js

```diff
@@ -66,6 +66,7 @@ module.exports = {
     "react/no-unescaped-entities": 0,
     "react/display-name": 0,
     "react/jsx-no-comment-textnodes": 1,
+    "react/no-unknown-property": ["error", {ignore: ["test-id"]}],
```

Comment:
I added a `test-id` to a component for easier access in cypress but it turns out that's disallowed by this lint rule and, IMO, it should be allowed.